src/engraving/libmscore/textbase.cpp

```diff
@@ -2393,7 +2393,7 @@ void TextBase::writeProperties(XmlWriter& xml, bool writeText, bool /*writeStyle
     }
 }
 
-static constexpr std::array<Pid, 18> pids { {
+static constexpr std::array<Pid, 18> TBPids { {
```

Comment:
Better to rename this array because the current name isn't clear. I think TextBasePropertyId is a good  name in this case

Reply:
I'll fix that. Wanted to keep.is as short as possible, but yes, it is too short